packages/wallet/contract/src/walletFactory.js

```diff
@@ -101,7 +100,10 @@ export const start = async (zcf, privateArgs) => {
 
     /** @type {() => Promise<import('./smartWallet').SmartWallet>} */
     const maker = () =>
-      makeSmartWallet({ address, bank, myAddressNameAdmin }, shared);
+      makeSmartWallet({ address, bank }, shared).then(wallet => {
+        E(myAddressNameAdmin).update('depositeFacet', wallet.getDepositFacet());
```

Comment:
`void`?

Reply:
not void. it's returning a promise for a smart wallet. up to the consume to handle rejection.

Reply:
`E(nameAdmin).update()` is returning the smart wallet? I don't see how. Maybe I'm distracted...

Reply:
the line after it is